ParkingLotApi/Services/OrderService.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices.ComTypes;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using ParkingLotApi.Dtos;
+using ParkingLotApi.Entities;
+using ParkingLotApi.Repository;
+
+namespace ParkingLotApi.Services
+{
+    public interface IOrderService
+    {
+        public Task<(OrderDto, string)> AddAsync(OrderCreateDto orderCreateDto);
+        public Task<OrderDto> GetAsync(string orderNumber);
+        public Task<OrderEntity> GetInDevAsync(string orderNumber);
+        public Task<List<OrderEntity>> GetAllInDevAsync();
+        public Task UpdateAsync(string orderNumber, OrderUpdateDto orderUpdateDto);
+    }
+
+    public class OrderService : IOrderService
+    {
+        private readonly ParkingLotContext parkingLotContext;
+
+        public OrderService(ParkingLotContext parkingLotContext)
+        {
+            this.parkingLotContext = parkingLotContext;
+        }
+
+        public async Task<(OrderDto, string)> AddAsync(OrderCreateDto orderCreateDto)
+        {
+            var parkingLot = this.parkingLotContext.ParkingLots
+                .FirstOrDefault(_ => _.Name == orderCreateDto.ParkingLotName);
+            if (parkingLot == null)
+            {
+                return (null, "the parking lot not found");
```

Comment:
return only one content according to SOLID?

Reply:
> return only one content according to SOLID?

yeah, may have some better methods to return error messages